### ISSUE TITLE
Addinng movie rating

### ISSUE THREAD

**Issue by TCarneiro91** [afonsojramos/discrakt#7]
Is it possible to get the imdb rating on the movie?

![image](https://user-images.githubusercontent.com/17798233/148684068-9a8edc7b-efec-4510-affa-ef503c3002fc.png)

**Comment by afonsojramos** [comment]
I don't think it makes much sense to include it in the status, as it would be yet another API call... What about a button to the IMDB page? Maybe that would work a bit better and I'm almost pretty sure that it might work: https://qwertyquerty.github.io/pypresence/html/doc/presence.html

**Comment by afonsojramos** [comment]
I'm still finishing the implementation of this. Will probably implement some caching to reduce calls to the Trakt API.
